spynnaker/pyNN/models/common/neuron_recorder.py

```diff
@@ -845,11 +954,13 @@ def get_sampling_overflow_sdram(self, vertex_slice):
             # If rate is 1 there is no overflow as average is exact
             if rate > 1:
                 data_size = self.get_buffered_sdram_per_record(
-                    variable,  vertex_slice)
+                    variable, vertex_slice, max_rewires_per_ts)
                 overflow += data_size // rate * (rate - 1)
         return overflow
 
-    def get_buffered_sdram(self, variable, vertex_slice, n_machine_time_steps):
+    def get_buffered_sdram(
+            self, variable, vertex_slice, n_machine_time_steps,
+            max_rewires_per_ts):
```

Comment:
If you can make this an instance variable (see elsewhere) it won't need passing in here.